.github/actions/get-latest-and-previous-tags-v1/action.yml

```diff
@@ -0,0 +1,23 @@
+name: get-latest-and-previous-tags-v1
+description: 'A GitHub Action to retrieve the latest and the previous one Git tags'
+
+outputs:
+  latest-tag:
+    description: 'Latest Git tag name (eg. v1.2.0)'
+    value: ${{ steps.get-tags.outputs.latest-tag }}
+  previous-tag:
+    description: 'Previous Git tag name (eg. v1.1.0)'
+    value: ${{ steps.get-tags.outputs.previous-tag }}
+
+runs:
```

Comment:
Do we expect this action to be used in more than place? If not, then this can be inlined as a step in the action consuming this vs being a reusable composite action